tests/phpunit/tests/Story_Renderer/HTML.php

```diff
@@ -90,6 +90,33 @@ public function test_replace_html_head() {
 		$this->assertSame( 1, did_action( 'web_stories_story_head' ) );
 	}
 
+	/**
+	 * @covers ::replace_html_head
+	 * @covers ::get_html_head_markup
+	 */
+	public function test_replace_html_head_invalid() {
+		$start_tag = '<meta name="web-stories-replace-head-start " />';
```

Comment:
Don't need to cater for whitespace within the attribute value
```suggestion
		$start_tag = '<meta name="web-stories-replace-head-start" />';
```

Reply:
Again, not a mistake, valid test case.